src/drawing/strokeactions.ts

```diff
@@ -72,7 +78,21 @@ export async function registerLiveStroke(pageId: string): Promise<void> {
         return
     }
 
-    handleAddStroke(createStroke(liveStroke, pageId, true))
+    const stroke = createStroke(liveStroke, pageId, true)
+
+    if (liveStroke.type === toolType.SELECT) {
+        const plen = stroke.points.length
+        const x1 = stroke.points[0]
+        const y1 = stroke.points[1]
+        const x2 = stroke.points[plen - 2]
+        const y2 = stroke.points[plen - 1]
+        store.dispatch(SELECT({ pageId, x1, y1, x2, y2, trRef, layerRef }))
```

Comment:
pass `points` directly with this action and handle x1,x2,... in the reducer, to avoid this boilerplate code when calling